src/applications/FlowrPhone/phoneWindow.ts

```diff
@@ -34,6 +35,65 @@ export class PhoneWindow extends KeyboardMixin(BrowserWindow) {
   private readonly _ipcEvents: {[key: string]: (...args: any[]) => void}
   private logger = getLogger('Phone window')
 
+  constructor(parent: BrowserWindow, preload: string | undefined, index: string, props: PhoneAppProps, private store?: Store<Record<string, any>> | undefined) {
```

Comment:
Not mandatory, like that you know. You can do something like that
{phoneMessagingNumber, phoneServer, ...}: PhoneProps to avoid doing 
props.phoneMessaging ... props.phoneServer.

But don't change anything

https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgJIBUCeAHFBvAKGWORmAgBsATARgC5kBnMKUAcyJLMqoCYHmrEBxKly1AMwCW7AgF8CBBAHsQzZNijLsjBhhwoAvMkKju1esgDk52lYA0nYrf7WXDp2J5S34qhKt5RRU1MFIAVxAEZGMACgo4ACNKaSE2exNbGgyXHL8JOT0sXABKGIA+E08QxmUKCAA6CmU2eKTKEurVWvqmltiAAyyGABI8LLkBztEausbm1qG-VzGXSemSWd6FwdsfVfz1oIIYSIRYqzAAC2BGVEYAQQAZdooHDS0dEqA

Reply:
That may be true, but in this case this would only make the constructor way longer for no added benefit below